docs/Installing.asciidoc

```diff
@@ -23,26 +23,33 @@ Installation
 ------------
 [id="installation"]
 
-The easiest way to install openQA is from packages. You can find them in OBS in
-the https://build.opensuse.org/project/show/devel:openQA[openQA] repository or
+The easiest way to install openQA is from packages. You can find openSUSE
+packages in OBS in the
+https://build.opensuse.org/project/show/devel:openQA[openQA] repository or
```

Comment:
I don't think I actually changed any of the SUSE repo text there, I only changed "them" to "openSUSE packages" and re-wrapped. So if there's an error, it was already there...